### PR TITLE
selenium-standalone: 7.0

### PR DESCRIPTION
**Pull request by mgrybyk**
As the only maintainer of `selenium-standalone`, I'd like to release types for the new major version 7.0

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/vvo/selenium-standalone/releases/tag/v7.0.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
Usually it would be better for actual maintainer to do a PR and remove herself/himself from the list, e.g.:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53883
Also the maintainer list is autogenerated file, should not be updated directly.
So I'd suggest to update the header just by adding your handle into the list

**Comment by mgrybyk** [comment]
@peterblazejewicz will do

File: types/selenium-standalone/tsconfig.json
Comment (peterblazejewicz) [inline types/selenium-standalone/tsconfig.json:5]: why this is required?

Comment (mgrybyk) [reply types/selenium-standalone/tsconfig.json:5]: Not at all.
The project has dropped NodeJS 10 support and now the minimum required is 12 that supports `es2019`.

Safe to remove if you prefer

Comment (mgrybyk) [reply types/selenium-standalone/tsconfig.json:5]: Just verified, lint and test passing with `es6`.
Shall I revert this change?

File: .github/CODEOWNERS
Comment (peterblazejewicz) [inline .github/CODEOWNERS]:
```suggestion
/types/selenium-standalone/                             @mgrybyk
```
so actually, reverts this line as outcome

File: types/selenium-standalone/tsconfig.json
Comment (peterblazejewicz) [reply types/selenium-standalone/tsconfig.json:5]: Yes, I'd say so. My concern was DT package interop with other packages, nothing specific.

File: .github/CODEOWNERS
Comment (mgrybyk) [reply .github/CODEOWNERS]: It was already changed, refresh the page

File: types/selenium-standalone/tsconfig.json
Comment (mgrybyk) [reply types/selenium-standalone/tsconfig.json:5]: Ok

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@mgrybyk please wait for original owner feedback, if possible!
thx!

**Comment by mgrybyk** [comment]
Ready to merge

**Pull request merged**
